src/app/views/dashboard/games/manage/game/packages/list/list.html

```diff
@@ -134,26 +138,30 @@
 							<app-jolticon icon="cog" />
 							<translate>Manage</translate>
 						</router-link>
-						<router-link
-							class="btn btn-outline"
-							:to="{
-								name: 'dash.games.manage.game.packages.edit.widget',
-								params: { packageId: package.id },
-							}"
-							>
-							<app-jolticon icon="plug" />
-							<translate>Widget</translate>
-						</router-link>
-						<router-link
-							class="btn btn-outline"
-							:to="{
-								name: 'dash.analytics',
-								params: { resource: 'Game_Package', resourceId: package.id },
-							}"
-							>
-							<app-jolticon icon="chart" />
-							<translate>Analytics</translate>
-						</router-link>
+						<app-game-perms required="all">
```

Comment:
we could probably let anyone see the widget. you can't modify anything on that page